python/b2gedmntuples_cff.py

```diff
@@ -1178,23 +1189,29 @@
 subjetsAK8CHS = copy.deepcopy(basic)
 subjetsAK8CHS.variables += jetVars
 subjetsAK8CHS.prefix = cms.untracked.string("subjetAK8CHS")
-subjetsAK8CHS.src = cms.InputTag("slimmedJetsAK8PFCHSSoftDropPacked", "SubJets")
+subjetsAK8CHS.src = cms.InputTag("selectedPatJetsAK8PFCHSSoftDropPacked", "SubJets")
 subjetKeysAK8CHS = copy.deepcopy( jetKeys )
-subjetKeysAK8CHS.jetLabel = cms.InputTag("slimmedJetsAK8PFCHSSoftDropPacked", "SubJets")
+subjetKeysAK8CHS.jetLabel = cms.InputTag("selectedPatJetsAK8PFCHSSoftDropPacked", "SubJets")
 
 ###subjetsAK8Puppi
-subjetsAK8Puppi = subjetsAK8CHS.clone( prefix = 'subjetAK8Puppi', 
-    src = cms.InputTag('selectedPatJetsAK8PFPuppiSoftDropPacked', "SubJets") 
-    )
+subjetsAK8Puppi = subjetsAK8CHS.clone( 
+		prefix = 'subjetAK8Puppi', 
+		src = cms.InputTag('selectedPatJetsAK8PFPuppiSoftDropPacked', "SubJets") )
 subjetKeysAK8Puppi = subjetKeysAK8CHS.clone( jetLabel = cms.InputTag('selectedPatJetsAK8PFPuppiSoftDropPacked', "SubJets") )
 
 ###subjetsAK8 CMSTopTag
 subjetsCmsTopTag = copy.deepcopy(basic)
 subjetsCmsTopTag.variables += jetVars
 subjetsCmsTopTag.prefix = cms.untracked.string("subjetCmsTopTag")
-subjetsCmsTopTag.src = cms.InputTag("slimmedJetsCMSTopTagCHSPacked", "SubJets")
+subjetsCmsTopTag.src = cms.InputTag("patJetsCMSTopTagCHSPacked", "SubJets")
 subjetsCmsTopTagKeys = copy.deepcopy( jetKeys )
-subjetsCmsTopTagKeys.jetLabel = cms.InputTag("slimmedJetsCMSTopTagCHSPacked", "SubJets")
+subjetsCmsTopTagKeys.jetLabel = cms.InputTag("patJetsCMSTopTagCHSPacked", "SubJets")
+
+###subjetsAK8 CMSTopTag
+subjetsCmsTopTagPuppi = subjetsCmsTopTag.clone( 
+		prefix = "subjetCmsTopTagPuppi",
+		src = cms.InputTag("patJetsCMSTopTagPuppiPacked", "SubJets") )
+subjetsKeysCmsTopTagPuppiKeys = subjetsCmsTopTagKeys.clone( jetLabel = cms.InputTag('patJetsCMSTopTagPuppiPacked', "SubJets") )
 
```

Comment:
I am overall in favour of dropping CMSToptagger jets, it will also remove the issue with the topMass and wMass variables.